bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveCommandClass.java

```diff
@@ -438,7 +442,10 @@ protected int extractValue(byte[] buffer, int offset, int size) {
 		SILENCE_ALARM(0x9D,"SILENCE_ALARM",null),
 		SENSOR_CONFIGURATION(0x9E,"SENSOR_CONFIGURATION",null),
 		MARK(0xEF,"MARK",null),
-		NON_INTEROPERABLE(0xF0,"NON_INTEROPERABLE",null);
+		NON_INTEROPERABLE(0xF0,"NON_INTEROPERABLE",null),
+		// USE MANUFACTURER SPECIFIC (0x91 [MANUFACTURER_PROPRIETARY] + MANUFACTURER_ID + DEVICE_TYPE_ID)
+		// 0x91 + 0x010F + 0x0301 for FGRM_222
+		FIABRO_FGRM_222(0x4A1,"FIABRO_FGRM_222",FibaroFGRM222CommandClass.class);
```

Comment:
I think I would remove this and we treat it different (somehow). The MANUFACTURER_ID + DEVICE_TYPE_ID is not strong enough to avoid errors.
I would suggest adding a separate list specifically for MANUFACTURER_PROPRIETARY that has the manufacturer, and type, and the class reference.